doc/modules/preprocessing.rst

```diff
@@ -336,6 +336,10 @@ lognormal distribution to a normal distribution::
          [-0.05...,  0.58..., -0.57...],
          [ 0.69..., -0.84...,  0.10...]])
 
+While the above example sets the `standardize` option to `False`, by default,
```

Comment:
To reduce syntactic ambiguity, put "by default" after "will".

Reply:
So close :)